File: abc.py
import os, sys

class Hello:
	def __init__(self, name):
		self.name=name
	def waveHand(self):
		print("{} is waving hands.".format(self.name))
	def sayHi(self):
		print("{} is saying hi.".format(self.name))

if __name__=='__main__':
	print('Hello')
	print(os.listdir())
	print(sys.path)
	h = Hello('Kathy')
	h.waveHand()
	h.sayHi()

File: abc.cpp
#include <iostream>
using namespace std;
int add(int, int);

int main() {
	int a=10, b=20;
	cout << "Hello World" << endl;
	cout << "a + b = " << add(a, b) << endl;
	return 0;
}


int add(int a, int b) {
	return a+b;
}
